external/audiocodec/streaming/player.c

```diff
@@ -29,615 +29,555 @@
 #include <assert.h>
 #include <pthread.h>
 #include <sys/time.h>
-#include "internal_defs.h"
-#include "debug.h"
+#include <debug.h>
 #include <audiocodec/streaming/player.h>
+#include "internal_defs.h"
 
-#define PV_SUCCESS 	0
-#define PV_FAILURE 	-1
 
-#define PLAYER_DEBUG 	audvdbg
-#define PLAYER_ERROR 	auddbg
-#define PLAYER_ASSERT 	MY_ASSERT
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define PV_SUCCESS  0
+#define PV_FAILURE  -1
 
-//#define PERFORMANCE_TEST
+#define PLAYER_DEBUG    medvdbg
```

Comment:
If you define MACRO with additional info, then it is ok.
But currently I couldn't find any advantage of this macros.
Actually medvdbg or meddbg displays function name..